### PR TITLE
updating tunable laser states

### PR DESCRIPTION
**Pull request by parfa30**
Associated with https://github.com/lsst-ts/ts_TunableLaser/pull/52

### CODE REVIEW

File: doc/news/DM-44083.misc.rst
Comment (couger01) [inline doc/news/DM-44083.misc.rst:1]: Needs to be under interface_changes folder and using the name DM-44083.tunablelaser.rst.

**Review by couger01 — CHANGES_REQUESTED** [review]
You need to update the enumeration here as well https://github.com/lsst-ts/ts_xml/blob/6b20a718d370e90034a02b34f6624f41a3a88269/python/lsst/ts/xml/data/sal_interfaces/TunableLaser/TunableLaser_Events.xml#L4. Also you need to correct the name of the news fragment but otherwise looks good.

File: doc/news/DM-44083.misc.rst
Comment (parfa30) [reply doc/news/DM-44083.misc.rst:1]: I hadn't realized that. Thanks! I moved and renamed the file.

Comment (couger01) [reply doc/news/DM-44083.misc.rst:1]: It seems like the old file wasn't removed, could you remove this one please?

**Review by couger01 — CHANGES_REQUESTED** [review]
Can you remove the old news fragment otherwise looks real good to me.

**Comment by parfa30** [comment]
Oh shoot! I thought it was removed. It should be now.

**Review by couger01 — APPROVED** [review]
LGTM, make sure to rebase on latest develop and squash the commits please.

**Pull request merged**